nextstrain/cli/command/deploy.py

```diff
@@ -1,87 +1,12 @@
-"""
-Deploys a set of built pathogen JSON data files or Markdown narratives to a
-remote location.
-
-nextstrain.org, or other instances of the Nextstrain web frontend (auspice),
-fetch the deployed JSON data files or Markdown narratives for display.
- 
- 
-Destinations
-------------
-
-Currently only Amazon S3 buckets (s3://…) are supported as the remote
-destination, but others can be added in the future.
-
-Destination URLs support optional path prefixes if you want your local
-filenames to be prefixed on the remote destination.  For example:
-
-    nextstrain deploy s3://my-bucket/some/prefix/ auspice/zika*.json
-
-will upload files named "some/prefix/zika*.json".
- 
- 
-Authentication
---------------
-
-Credentials for authentication should generally be provided by environment
-variables specific to each destination type.
-
-S3
---
-
-* AWS_ACCESS_KEY_ID
-* AWS_SECRET_ACCESS_KEY
-
-More information at:
-
-    https://boto3.readthedocs.io/en/latest/guide/configuration.html#environment-variables
-
-A persistent credentials file, ~/.aws/credentials, is also supported:
-
-    https://boto3.readthedocs.io/en/latest/guide/configuration.html#shared-credentials-file
- 
-"""
-
-from pathlib import Path
-from urllib.parse import urlparse
-from ..util import warn
-from ..deploy import s3
-
-
-SUPPORTED_SCHEMES = {
-    "s3": s3,
-}
+# This command, `nextstrain deploy`, is now an alias for `nextstrain remote
+# upload`.
+#
```

Comment:
Makes sense to not duplicate this code, but it results in a help message for `nextstrain deploy` which talks about `nextstrain remote upload`. The help should be amended so that it either states that this is an alias for `nextstrain remote upload` or to indicate that it's deprecated and users should move to `nextstrain remote upload`.